cmd/brew-test-bot.rb

```diff
@@ -595,7 +595,10 @@ def setup
       @category = __method__
       return if @skip_setup
       # install newer Git when needed
-      test "brew", "install", "git" if OS.mac? && MacOS.version < :sierra
+      if OS.mac? && MacOS.version < :sierra
+        test "brew", "install", "git"
+        ENV["HOMEBREW_FORCE_BREWED_GIT"] = "1"
```

Comment:
If the host `git` doesn't exist or is too old, it'd be great if `HOMEBREW_FORCE_BREWED_GIT` were the default behaviour.

Reply:
I think this _could_ be related to upstream as well.

Reply:
I'd meant to direct the above comment to @MikeMcQuaid. It's been on my wish list for a while but I haven't gotten to it yet.

Reply:
@sjackman It should be the case already when it's too old. I'm not sure why `HOMEBREW_FORCE_BREWED_GIT` is required here but I think it could be because `brew test-bot` is itself a `brew` process that's set `*GIT*` stuff.

Reply:
(or because it cleaned up the Git it installed already and then the environment variable doesn't behave correctly)